### PR TITLE
Alto-1654: Readme updates

### PR DESCRIPTION
**Pull request by ShlomiAltostra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,92 @@
...
-The template is made of two resources: 
+## Getting started
+
+### Things you'll need 
+1. AWS [SAM-CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+1. An Altostra Account( Don't have one yet? Just [login](https://app.altostra.com) here)
+1. Altostra CLI Installed ([how?](../reference/CLI/altostra-cli.html#installation))
+1. VSCode with Altostra Editor ([how?](../getting-started/installation.html#install-the-visual-studio-code-extension))
+1. A Connected AWS Account ([how?](../getting-started/connect-your-accounts.html#connect-your-cloud-service-accounts))
+1. An Environment connected to your AWS Account ([how?](../howto/envs/manage-environments.html)) - We'll call it `Dev` for berevity, but you can pick any of your environments
```
Comment (yevk) [inline README.md]: Update with latest from Yossi, he changed it a bit.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,92 @@
...
+
+### Using the template
+
+You have several options to get started with this template.  
+Either go to the Altostra Web Console and create a new project.  
+When asked to use a template, select "Static Website".
```
Comment (yevk) [inline README.md]: It doesn't show up like that, it shows up as `static-website`.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,92 @@
...
+
+Alternatively, you can use the Altostra CLI to initialize a new project from the template by running:
+```sh
+$ alto init --template static-website
+```
+
+You can also apply the template to an existing Altostra project from Visual Studio Code by going 
+to the Altostra view in the main toolbar and clickign on "Static Website" in the templates list.
```
Comment (yevk) [inline README.md]: Make sure it appears that way.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,92 @@
...
+
+### Required configuration
+
+This template contains a few parameters that must be 
+[set in the the environment](https://docs.altostra.com/howto/envs/manage-environments.html#manage-configuration-values)
+in which the deployment was created.
```
Comment (yevk) [inline README.md]:
```suggestion
This template contains a few parameters that must be 
[set in the environment](https://docs.altostra.com/howto/envs/manage-environments.html#manage-configuration-values) to which the project will be deployed.
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,92 @@
...
+
+You may also provide all or part of the configuration when 
+[creating or updating the deployment](https://docs.altostra.com/reference/CLI/commands/altostra-cli-deploy.html#providing-configuration).
```
Comment (yevk) [inline README.md]: Let's keep it simple and not give them all the options here in the readme.
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,12 +1,92 @@
...
+
+- `DOMAIN`:  
+The fully qualified root domain of your application or arganization.
+- `DOMAIN_HOST_NAME`:  
+A host name for the site CDN.  
+It is combined with `DOMAIN` to create a fully qualified domain for the CDN.  
+I.E. If we set `DOMAIN: example.com` and `DOMAIN_HOST_NAME: my-site` the CDN would
+accept requests for `my-site.example.com`.
+- `DOMAIN_CERT`:  
+An Amazon certificate manager (ACM) certificate ARN approprite for the domain.
```
Comment (yevk) [inline README.md]: I think you can remove this after Yossi's change.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,92 @@
...
+
+### Project deployment
+
+Here are simple instruction to get you started.  
+For more information on each command please consult [Altostra docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).
```
Comment (yevk) [inline README.md]:
```suggestion
Run the following commands to create a deployment image of the project and deploy it as a new instance.  
For more information on each command refer to the [Altostra CLI docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).
```

File: README.md [hunk 8]
```diff
@@ -1,12 +1,92 @@
...
+
+1. Create an image from the project:
```
Comment (yevk) [inline README.md]: Link the word _Image_ to the docs.

File: README.md [hunk 9]
```diff
@@ -1,12 +1,92 @@
...
+```shell
+$ altos push v1.0
+```
+2. Deploy the image to a new deployment named `main` in the `Dev` environment:
```
Comment (yevk) [inline README.md:40]: You can't just throw the reader at the Dev environment if it doesn't exist. Explain that and/or provide instructions to create one.

File: README.md [hunk 10]
```diff
@@ -1,12 +1,92 @@
...
+```shell
+$ alto deploy main:v1.0 --new Dev
+```
+3. Update the deployment with new images:
+```shell
+$ alto deploy main:v2.0
```
Comment (yevk) [inline README.md]: No need for an update if nothing has changed. You make it look like it's part of the required steps.
A better step would be to run `alto deployments main`.

File: README.md [hunk 11]
```diff
@@ -1,12 +1,92 @@
...
+```
+4. Manage the project in the Altostra Web Console:
+```shell
+$ alto console
+```
+
+### Upload and invalidate site content:
+
+1. Upload content of all S3 buckets of `main` deployment:
```
Comment (yevk) [inline README.md:60]: Add a text before this to explain why these steps are necessary, that the content upload is split from the infrastructure deployment.

File: README.md [hunk 12]
```diff
@@ -1,12 +1,92 @@
...
+```shell
+$ alto invalidate main --all
+```
+
+## Content
+
+The template is made of three resources: 
 - An S3 Bucket
 - A CloudFront CDN
 - Domain Name
```
Comment (yevk) [inline README.md]: Update to Yossi's changes.

**Review by yevk — CHANGES_REQUESTED** [review]
I think that, maybe, some of the missing info can be located in the docs website.

File: README.md (hunk 9 above)
Comment (ShlomiAltostra) [reply README.md:40]: I can
It is in the prerequisites list, with instructions

File: README.md [hunk 13]
```diff
@@ -3,73 +3,61 @@ Use this template to create a static website that is published over a CDN.
...
 ```shell
 $ alto console
 ```
 
+> To update an existing deployment with new images just omit the `--new` flag and environment name:
+> ```shell
+> $ alto deploy main:v2.0
+>```
+
 ### Upload and invalidate site content:
```
Comment (yevk) [inline README.md]:
```suggestion
### Upload and invalidate site content
```

**Review by yevk — CHANGES_REQUESTED** [review]
Small change and a big one: Add the command to get domains and tell reader to open that URL

**Review by yevk — APPROVED** [review]

**Pull request merged**